modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductSpecificationResourceImpl.java

```diff
@@ -52,6 +54,23 @@
 public class ProductSpecificationResourceImpl
 	extends BaseProductSpecificationResourceImpl implements NestedFieldSupport {
 
+	@Override
+	public Response deleteProductSpecification(Long id) throws Exception {
```

Comment:
delete method should return void

Reply:
This is done like all the other deletion 

Example: public Response deleteSku(Long id) throws Exception {

Reply:
Changed as suggested. I did it "in the old way"